arrow/arrow.py

```diff
@@ -295,6 +295,34 @@ def strptime(cls, date_str, fmt, tzinfo=None):
             fold=getattr(dt, "fold", 0),
         )
 
+    @classmethod
+    def fromordinal(cls, ordinal):
+        """Constructs an :class:`Arrow <arrow.arrow.Arrow>` object corresponding
+            to the Gregorian Ordinal.
+
+        :param ordinal: an ``int``.
```

Comment:
Maybe change this comment to "an ``int`` representing an Gregorian Ordinal`

Reply:
I agree, I will change it.